logging/google/cloud/logging/_gax.py

```diff
@@ -418,11 +424,14 @@ def metric_update(self, project, metric_name, filter_, description):
         metric_pb = LogMetric(name=path, filter=filter_,
                               description=description)
         try:
-            self._gax_api.update_log_metric(path, metric_pb, options=options)
+            metric_pb = self._gax_api.update_log_metric(
```

Comment:
Likewise, `LogMetric` has no output-only fields.  It can't *hurt* to capture it, but it shouldn't matter, either.